src/app/hooks/useTransactions.ts

```diff
@@ -11,22 +11,29 @@ export const useTransactions = () => {
   const [isLoadingTransactions, setIsLoadingTransactions] = useState(true);
 
   const loadTransactions = useCallback(
-    async (accountId: string, limit?: number) => {
+    async (limit?: number) => {
       try {
-        const { payments } = await api.getPaymentsByAccount({
-          accountId,
+        const getTransactionsResponse = await api.getTransactions({
+          isSettled: true,
           limit,
         });
-        const _transactions: Transaction[] =
-          await convertPaymentsToTransactions(payments);
 
-        for (const transaction of _transactions) {
+        const transactions = getTransactionsResponse.transactions.map(
```

Comment:
Do we still need to do the mapping here, or should this be handled somewhere else?